packages/actions/index.ts

```diff
@@ -1143,3 +1160,64 @@ export const getTransferLimitLevel = async (userAddress, store) => {
 export const exceedsTransferLimit = (amount: number, transferLimit: string, ethExchange: string, ethSentPastWeek: number) => {
   return (ethSentPastWeek * Number(ethExchange) + amount) > Number(transferLimit)
 }
+
+export const sendEmailTx = (inviteTx: InviteTransaction) => {
+  return async (dispatch, getState) => {
+    const { privateKeyBuffer } = getUser(getState())()
+
+    return creditProtocol.sendEmailTx(inviteTx, privateKeyBuffer)
+    .catch(err => {
+      dispatch(displayError(debtManagement.pending.error))
+      throw new Error('Unable to send email transaction: ' + err)
+    })
+  }
+}
+
+export const getEmailTx = (hash: string) => {
+  return creditProtocol.getEmailTx(hash)
+}
+
+  // return creditProtocol.deleteEmailTx(hash)
```

Comment:
:fire: Generally, all commented-out lines or blocks of code should be removed. Keeping them in, they slowly drift out of relevance and begin to cause ":confused: ...hmm, what is this commented-out code all about. What does it do again?"